packages/services/test/kernel/ikernel.spec.ts

```diff
@@ -660,7 +660,8 @@ describe('Kernel.IKernel', () => {
       await expectFailure(restart);
     });
 
-    it('should dispose of existing comm and future objects', async () => {
+    // TODO: enable this test when the CI has upgraded to xeus-python 0.12.3
```

Comment:
Just to double check: is this test constantly failing, or was maybe just flaky?

Reply:
I've added the test back, let's see.

Reply:
Now passing, so my assumption was wrong. I will rebase and drop the last two commits.